### PR TITLE
rtc docs hotfixes

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Various minor hotfixes to RTC docs

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by JamesToohey — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

File: _includes/configuration/rtc_client_connected.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Required plugin
 {% endif %}
 
 Type
-: Function (Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise))
```
Comment (lnewson) [inline _includes/configuration/rtc_client_connected.md:15]: There's quite a number that have been missed here:

Comment (lnewson) [reply _includes/configuration/rtc_client_connected.md:15]: So looking at that list, only 4 out of 8 have been changed.

**Review by lnewson — APPROVED** [review]
Looks fine except for the 4 types that look like they still need fixing.

File: _includes/configuration/rtc_client_connected.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/configuration/rtc_client_connected.md:15]: those are promises. these ones are callbacks

Comment (TheSpyder) [reply _includes/configuration/rtc_client_connected.md:15]: Yes the problem is it was added to all 8 functions, but only 4 of them needed it. I didn't catch this in the pre-release review.

**Review by ashfordneil — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Review by lostkeys — APPROVED** [review]

**Pull request merged**